src/interactive/HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+0.5.0
++++++
+* Support command recommendations that predicts the next commands users might need.
+* Support scenario recommendations to help users complete the complex scenarios more smoothly and simply.
+* Support new config `az config set interactive.enable_recommender=False` to disable these new recommenders.
+* Fix display offset in toolbar.
+* Add loading bar to avoid command execution errors caused by uncompleted command loading and param update. User can use CTRL+C to cancel the loading bar.
+* Add value completion mechanism for param value in scenarios
+* Support searching ability for scenarios
+* Optimize param completion mechanism to recommend one parameter at a time
```

Comment:
Avoid to use abbreviation, so please use "parameter" instead of "param" to make it more clear to customers.

Reply:
Thank you for the suggestion, already modified in https://github.com/Azure/azure-cli-extensions/pull/5950/commits/127f75bc3513400256777af1728dd650a8115f1f